packages/synapse-react-client/src/components/SetAccessRequirementCommonFields/SetAccessRequirementCommonFields.tsx

```diff
@@ -199,26 +208,32 @@ export const SetAccessRequirementCommonFields = React.forwardRef(
             const isStillLoading =
               (isEditing && !accessRequirement) || !subjectsType
             const hasSubjectsError = hasPendingSubjects || subjects.length === 0
-
-            if (isStillLoading || hasSubjectsError) {
-              if (hasSubjectsError && !isStillLoading) {
-                if (hasPendingSubjects) {
-                  setSubjectsError(UNSAVED_SUBJECTS_ERROR_MESSAGE(subjectsType))
-                } else if (subjects.length === 0) {
-                  setSubjectsError(EMPTY_SUBJECT_LIST_ERROR_MESSAGE)
+            if (!subjectsDefinedByAnnotations) {
+              if (isStillLoading || hasSubjectsError) {
+                if (hasSubjectsError && !isStillLoading) {
+                  if (hasPendingSubjects) {
+                    setSubjectsError(
+                      UNSAVED_SUBJECTS_ERROR_MESSAGE(subjectsType),
+                    )
+                  } else if (subjects.length === 0) {
+                    setSubjectsError(EMPTY_SUBJECT_LIST_ERROR_MESSAGE)
+                  }
                 }
-              }
 
-              onError()
-              return
+                onError()
+                return
+              }
             }
-
+            const newAccessType = subjectsDefinedByAnnotations
+              ? ACCESS_TYPE.DOWNLOAD
+              : getAccessType(subjects[0])
```

Comment:
This is a change I should've made originally, but since we've already stored the AR's `RestrictableObjectType` in `subjectsType` , could `getAccessType` be refactored to return the `ACCESS_TYPE` given the `RestrictableObjectType` rather than given an `RestrictableObjectDescriptor`? Then I think this could be simplified to -- 

```suggestion
            const newAccessType = getAccessType(subjectsType)
```

Reply:
Done!